bdns/dns_test.go

```diff
@@ -178,19 +179,35 @@ func mockDNSQuery(w dns.ResponseWriter, r *dns.Msg) {
 
 func serveLoopResolver(stopChan chan bool) {
 	dns.HandleFunc(".", mockDNSQuery)
-	server := &dns.Server{Addr: dnsLoopbackAddr, Net: "tcp", ReadTimeout: time.Second, WriteTimeout: time.Second}
+	tcpServer := &dns.Server{
+		Addr:         dnsLoopbackAddr,
+		Net:          "tcp",
+		ReadTimeout:  time.Second,
+		WriteTimeout: time.Second,
+	}
+	udpServer := &dns.Server{
+		Addr:         dnsLoopbackAddr,
+		Net:          "udp",
+		ReadTimeout:  time.Second,
+		WriteTimeout: time.Second,
+	}
+	go func() {
+		err := tcpServer.ListenAndServe()
+		if err != nil {
+			log.Fatal(err)
+		}
+	}()
 	go func() {
-		err := server.ListenAndServe()
+		err := udpServer.ListenAndServe()
 		if err != nil {
-			fmt.Println(err)
-			return
+			log.Fatal(err)
 		}
 	}()
 	go func() {
 		<-stopChan
 		err := server.Shutdown()
```

Comment:
I think this needs to change to close `udpServer` and `tcpServer`. CI is failing right now with an error:
> bdns/dns_test.go:208:10: undefined: server